tests/unit/storage/metadata/mongoclient/ListRecordStream.spec.js

```diff
@@ -364,4 +364,42 @@ describe('mongoclient.ListRecordStream', () => {
             done();
         });
     });
+
+    it('should support tags with dots and dollars', done => {
+        const logEntry = {
+            h: -42,
+            ts: Timestamp.fromNumber(42),
+            op: 'i',
+            ns: 'some-bucket',
+            o: {
+                _id: 'replicated-key\u000098467518084696999999RG001  19.3',
+                value: {
+                    tags: {
+                        someKey: '\uFF041\uFF0E5',
```

Comment:
I though you had the issue specifically with tag keys, so `'\uFF041\uFF0E5': 'someValue'` instead

Reply:
the "tags" object is processed indifferently for keys or values. So you can have dots and dollars in keys too.